.zenodo.json

```diff
@@ -0,0 +1,51 @@
+{
+  "creators": [
+    {
+      "name": "Ehbrecht, Carsten",
+      "affiliation": "DKRZ"
+    },
+    {
+      "name": "Charette-Migneault, Francis",
+      "affiliation": "CRIM",
+      "orcid": "0000-0003-4862-3349"
+    },
+    {
+      "name": "Derue, François-Xavier",
+      "affiliation": "CRIM"
+    },
+    {
+      "name": "Smith, Trevor James",
+      "affiliation": "Ouranos",
+      "orcid": "0000-0001-5393-8359"
+    },
+    {
+      "name": "Byrns, David",
+      "affiliation": "Intact"
+    },
+    {
+      "name": "Perron, Louis-David",
+      "affiliation": "CRIM"
+    }
+  ],
+  "keywords": [
+    "security",
+    "ows",
+    "PAVICS",
+    "Weaver",
```

Comment:
I wouldn't consider Weaver here, although it is in the readme for historical reasons, they are not really connected anymore.

Preferable to focus on OGC API, or even, REST API in general.